### PR TITLE
Feat: transcript sentiment

### PR DESCRIPTION
**Pull request by getnashty**

### CODE REVIEW

File: src/modules/Transcript/index.tsx [hunk 1]
```diff
@@ -166,19 +169,37 @@ export const TranscriptUI = (props: TranscriptUIProps): ReactElement => {
...
+                                                                    <span
+                                                                        key={sentenceId}
+                                                                        className={classNames({
+                                                                            'text-green-600':
+                                                                                textSentiment === 'positive',
+                                                                            'text-red-600':
+                                                                                textSentiment === 'negative',
+                                                                        })}
+                                                                    >
+                                                                        {/(\.|\?|!)$/.exec(text) ? `${text} ` : text}
```
Comment (getnashty) [inline src/modules/Transcript/index.tsx:200]: wow this is some shit! in my code it was like text.match(/...) shit got completely re-written like I'm a baby or some shit

Comment (lamflam) [reply src/modules/Transcript/index.tsx:207]: I don't get it lol. What is this doing?

File: src/modules/Transcript/index.tsx [hunk 2]
```diff
@@ -34,17 +34,21 @@ import { useInterval } from '@aiera/client-sdk/lib/hooks/useInterval';
...
 import { Playbar } from '@aiera/client-sdk/components/Playbar';
 import { EmptyMessage } from './EmptyMessage';
 import { Header } from './Header';
 import './styles.css';
 
 type SpeakerTurn = TranscriptQuery['events'][0]['transcripts'][0]['sections'][0]['speakerTurns'][0];
 type Paragraph = SpeakerTurn['paragraphs'][0];
-type Chunk = { text: string; id: string; highlight: boolean };
+type Chunk = { text: string; id: string; highlight: boolean; textSentiment?: string };
+interface Sentences {
```
Comment (lamflam) [inline src/modules/Transcript/index.tsx:46]: I think this should be singular right?

**Review by lamflam — COMMENTED** [review]
Can you add a test that just checks that the sentiment markup is shown if and only if the sentiment values are on the sentences? I know it's annoying, but this component in particular keeps falling further behind in test coverage and I have to go back and add a ton already.

File: src/modules/Transcript/index.tsx (hunk 2 above)
Comment (getnashty) [reply src/modules/Transcript/index.tsx:46]: yup good call

**Review by lamflam — APPROVED** [review]

**Pull request merged**